Components/Avatar/Avatar.css

```diff
@@ -0,0 +1,33 @@
+/* -------------------- avatar----------------------- */
+
+/* ---------- profile avatar------------ */
+.avatar {
+    border-radius: 50%;
+    margin: 0 1rem;
+    object-fit: cover;
+    height: 9rem;
+    width: 9rem;
+    border: 2px solid var(--white);
+}
+
+
+/* -------------- sized avatar ---------- */
+.la {
+    height: 11rem;
+    width: 11rem;
+}
+
+.md {
+    height: 9rem;
+    width: 9rem;
+}
+
+.sm {
+    height: 5rem;
+    width: 5rem;
+}
+
+/* ---------------- square avatar--------- */
+.square {
```

Comment:
Same as above. Try looking into BEM naming convention 🤔

Reply:
changed the class name to sq-border as I felt it described what it does better